scripts/delete-bridges.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Script used to delete Bridges and Processors used in E2E tests
+# Script will delete Bridges without Processors, for Bridges with Processors it will delete Processors if they are in proper state
+# Test Bridges are identified using "test-" prefix in their names
+
+# If Bridge contains Processors then only Processors are deleted by the script execution. If user wants to delete also Bridges then the script has to be executed again after some time (so the Processors can be removed first)!
+
+BRIDGES=($(curl -s -H "Authorization: $OB_TOKEN" -X GET $MANAGER_URL/api/smartevents_mgmt/v1/bridges | jq '.items[] | select(.name | startswith('testui-$CYPRESS_SUITE_HASH')) | .id' | tr -d \"))
+
+for i in "${BRIDGES[@]}"
+do
+   NUMBER_OF_PROCESSORS=$(curl -s -H "Authorization: $OB_TOKEN" -X GET $MANAGER_URL/api/smartevents_mgmt/v1/bridges/$i/processors | jq '.items | length')
+  if [ "$NUMBER_OF_PROCESSORS" == "0" ]; then
+    echo "No processors in Bridge $i."
+    BRIDGE_STATUS=$(curl -s -H "Authorization: $OB_TOKEN" -X GET $MANAGER_URL/api/smartevents_mgmt/v1/bridges/$i | jq '.status' | tr -d \")
+    if [ "$BRIDGE_STATUS" == "ready" ] || [ "$BRIDGE_STATUS" == "failed" ]; then
+      echo "Deleting Bridge $i"
+      curl -s -H "Authorization: $OB_TOKEN" -X DELETE $MANAGER_URL/api/smartevents_mgmt/v1/bridges/$i
+    else
+      echo "Bridge $i status $BRIDGE_STATUS is incompatible with automated deletion, please remove it manually"
+    fi
+  else
+    echo "$NUMBER_OF_PROCESSORS processors found in Bridge $i"
```

Comment:
So far In our E2E tests we are not creating any processors, but in future we may add. 
In that case , we need to rework this script to delete bridge after processor deletion too.

Reply:
@Srihari1192 I will take a look at it, yesterday I have worked on the concept functionality. It seems that hash generated by CI is visible from Cypress tests and even if E2E failed than the clean hook is performed correctly. Delete all instances which contains "testui-$CYPRESS_SUITE_HASH".